trade-finance-manager-api/src/v1/controllers/auth/auth-provider-handleRedirect.api-test.js

```diff
@@ -0,0 +1,44 @@
+const AuthProvider = require('./auth-provider');
+
+describe('AuthProvider - handleRedirect', () => {
+  let result;
+
+  const mockOrigAuthCodeRequest = {
+    mockOriginalRequest: true,
+  };
+
+  const mockCode = 'mock-authz-code';
+
+  const mockPkceCode = {
+    verifier: 'mock-verifier',
+  };
+
+  const mockAcquireTokenByCodeResponse = {
+    account: {
+      idTokenClaims: [],
+    },
+  };
+
+  const mockMsalInstance = {
+    acquireTokenByCode: jest.fn().mockResolvedValue(mockAcquireTokenByCodeResponse)
+  };
+
+  beforeAll(async () => {
+    AuthProvider.getMsalInstance = () => mockMsalInstance;
+
+    result = await AuthProvider.handleRedirect(mockPkceCode, mockOrigAuthCodeRequest, mockCode);
```

Comment:
acquireTokenByCode should verify auth code and throw exception.

Can we add more tests where we throw mock  exceptions